cvmfs/mountpoint.h

```diff
@@ -399,6 +404,39 @@ class FileSystem : SingleCopy, public BootFactory {
   bool has_custom_sqlitevfs_;
 };
 
+/**
+ * The StatfsCache class is a class purely designed as "struct" (= holding
+ * object for all its parameters). 
+ * All its logic, including the locking mechanism, is implemented in the 
+ * function cvmfs_statfs in cvmfs.cc
+ */
+class StatfsCache {
```

Comment:
Let's make sure we cannot copy the class (since it maintains the mutex resource):
```suggestion
class StatfsCache : SingleCopy {
```